src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/jpa/StaffStatus.kt

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.digital.hmpps.nomisuserrolesapi.jpa
+
+enum class StaffStatus(
+  val code: Int,
+  val desc: String,
+) {
+  Active(0, "Active"),
```

Comment:
Bit confused with the Active and ACTIVE enums

Reply:
Active and ACTIVE both are status in Nomis. Just taken all statuses.

Reply:
@ravmoj need to check PROD -as not always the same as Dev

Reply:
Suggest if only in Dev we update the data

Reply:
Think this is reference data - can't really use Enum